Website/DesktopModules/Admin/Authentication/Login.ascx.cs

```diff
@@ -252,6 +252,8 @@ protected string RedirectURL
             }
         }
 
+        private bool NeedRedirectAfterLogin => LoginStatus == UserLoginStatus.LOGIN_SUCCESS;
```

Comment:
please take the following status into account,
LOGIN_SUPERUSER
LOGIN_INSECUREHOSTPASSWORD
LOGIN_INSECUREADMINPASSWORD